api/v1/serializers/developers-serializer.js

```diff
@@ -69,6 +69,6 @@ const serializeDeveloper = (rawDeveloper) => {
   return new JsonApiSerializer(
     developerResourceType,
     serializerOptions(serializerArgs),
-  ).serialize(rawDeveloper);
+  ).serialize([rawDeveloper]);
```

Comment:
My bad, I didn't catch this error yesterday. When sending a request to `GET /developers/45`, I got the following response:

```json
{
    "links": {
        "self": "https://localhost/v1/developers/45"
    },
    "data": [
        {
            "type": "developer",
            "id": "45",
            "links": {
                "self": "https://localhost/v1/developers/45"
            },
            "attributes": {
                "name": "Jagex",
                "website": "www.runescape.com"
            }
        }
    ]
}
```

The data field shouldn't be an array for a singleton resource. @Ruefa can you fix it?